tfrecorder/client_test.py

```diff
@@ -168,11 +175,28 @@ def test_gcs_path_with_dataflow_runner_missing_param(self):
             df2,
             runner='DataflowRunner',
             project=p,
-            region=r)
+            region=r,
+            tfrecorder_wheel=self.test_wheel)
       self.assertTrue('DataflowRunner requires valid `project` and `region`'
                       in repr(context.exception))
 
 
+  def test_gcs_path_with_dataflow_runner_missing_wheel(self):
+    """Tests DataflowRunner with missing required whl path."""
+    df2 = self.test_df.copy()
+    df2[constants.IMAGE_URI_KEY] = 'gs://' + df2[constants.IMAGE_URI_KEY]
+    with self.assertRaises(AttributeError) as context:
+      client._validate_runner(
+          df2,
+          runner='DataflowRunner',
+          project=self.test_project,
+          region=self.test_region,
+          tfrecorder_wheel=None)
+      print(context.exception)
```

Comment:
Maybe we should omit this print statement.

Reply:
Good catch, yes!